omnibus-base-sepolia-andromeda.toml

```
@@ -301,6 +326,44 @@ defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
 defaultValue = "<%= imports.erc_4626_to_assets_ratio_oracle.imports.statausdc_token_mock.contracts.Token.address %>"
 description = "We use the mock generated by erc-4626-to-assets-ratio-oracle package in statausdc-usdc.toml"
 
+[provision.cbbtc_mock_collateral]
+source = "mintable-token:1.8"
+target = "synthetix-mock-tokens:1.8@cbbtc"
+options.name = "Fake Coinbase Bitcoin"
+options.symbol = "fcbbtc"
```

to avoid diffs in tests, please use the same symbol for fake tokens as for real ones. So gonna be 
```suggestion
options.symbol = "cbBTC"
```
(same for other mocks)

---

hey @noisekit why are we dropping the `f`  for fake?

---

updated as well as others, but kept `f` can remove if you think worthwhile

---

Please remove `f`. That is pain to maintain separate symbols. Ideally e2e test should be pretty much exactly same as prod. And all configs as well. Only addresses different

---

ok will remove them nikita